torch/nn/quantized/modules/conv.py

```diff
@@ -75,7 +77,7 @@ def scale(self, s):
         if isinstance(s, torch.Tensor):
             self._scale = s
         else:
-            self._scale = torch.Tensor([s])
+            self._scale = torch.tensor([s], dtype=torch.double)
```

Comment:
Can we ensure that this is consistent across all quantized modules? @zafartahirov : Can you verify this?

Reply:
I can go through the modules to make sure this is the case. However, setting the tensor to double is optional (I think) -- it is the default.